app/lib/theme/ReadOnlyFieldTemplate.tsx

```diff
@@ -15,7 +15,7 @@ const FieldTemplate: React.FC<FieldTemplateProps> = ({
 }) => {
   return (
     <div>
-      <div className="definition-container">
+      <div className={`${uiSchema["ui:classNames"] ?? "definition-container"}`}>
         {displayLabel && (
```

Comment:
This is a great solution, nice work! I think it could be simplified to something like:

```<div className={ uiSchema.calculatedValueFormContextProperty ? "adjustable-calculated-value-widget":"definition-container"}`}>

Then any calculated field will get the style without us having to remember to add classnames to the uiSchema

Reply:
Great suggestion 🤩